### PR TITLE
Clarify language and grammar

### PR DESCRIPTION
**Pull request by mavaddat**

### CODE REVIEW

**Review by scop — CHANGES_REQUESTED** [review]

File: README.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ other file sourcing those). You can do this by simply using:
     . /usr/share/bash-completion/bash_completion
 ```
 
-(if you happen to have *only* bash >= 4.2 installed, see further if not)
+(above works if you have *only* bash >= 4.2 installed; see further if not)
```
[the remark this answers is not in the document]
Comment (mavaddat) [reply README.md:28]: Yes, sounds good. Clear.

[the remark this answers is not in the document]
Comment (scop) [reply README.md:28]: Can I ask you to update the PR with the discussed changes? Thanks!

[the remark this answers is not in the document]
Comment (mavaddat) [reply README.md:28]: Yes. Doing now.

**Comment by scop** [comment]
Addressed open conversations and merged as 6f1bbda3c66814befa8025d49363b4070ef20008.

**Pull request merged**